packages/roosterjs-editor-core/lib/corePlugins/TypeInContainerPlugin.ts

```diff
@@ -84,5 +101,28 @@ export default class TypeInContainerPlugin implements EditorPlugin {
                 }
             }
         }
+
+        /**
+         * Add a Span with default format to the previous element when pressing backspace
+         */
+        if (
+            event.eventType == PluginEventType.KeyDown &&
+            event.rawEvent.which == Keys.BACKSPACE &&
+            this.editor?.isFeatureEnabled(ExperimentalFeatures.DefaultFormatInSpan)
+        ) {
+            const element = this.editor?.getElementAtCursor();
+            const block =
+                element &&
+                this.editor?.getBlockElementAtNode(element)?.getStartNode().previousSibling;
+
+            if (block) {
+                this.editor?.runAsync(editor => {
+                    const position = editor.getFocusedPosition();
+                    if (position && block == position.element) {
+                        editor.ensureTypeInContainer(position, event.rawEvent);
```

Comment:
If user changed to another format, will this go back to default format?

Reply:
Yes, it will go back to default format, that seem to be the same behavior when the Experimental Feature is OFF.